testrail/client.py

```diff
@@ -172,13 +183,23 @@ def active_plans(self):
     # Run Methods
     @methdispatch
     def runs(self):
-        return map(Run, self.api.runs(self._project_id))
+        return RunContainer(list(map(Run, self.api.runs(self._project_id))))
 
     @runs.register(Milestone)
     def _runs_for_milestone(self, obj):
         return RunContainer(filter(
             lambda r: r.milestone.id == obj.id, self.runs()))
 
+    @runs.register(Active)
+    def _runs_for_active(self, obj):
```

Comment:
That is interesting, I wasn't aware of the 250 max.  That feels like a bug that should be reported to the testrail team as there needs to be a way to get all results.  I am open to adding params to the API calls, however I feel that a Filter class is overkill/unneeded and maybe even a little confusing.  We could just add the appropriate query params as arguments to the function.